apps/test/unit/templates/sectionProgressV2/SectionProgressSelectorTest.jsx

```diff
@@ -176,4 +188,37 @@ describe('SectionProgressSelector', () => {
     expect(screen.queryByText(V2_PAGE_LINK_TEXT)).to.not.exist;
     expect(screen.queryByTestId(V2_TEST_ID)).to.not.exist;
   });
+
+  it('calls analytics when toggling to v2', () => {
+    renderDefault();
+
+    const toggle = screen.getByRole('link', {name: V1_PAGE_LINK_TEXT});
+
+    fireEvent.click(toggle);
+
+    expect(postStub).to.have.been.calledTwice;
```

Comment:
I do think this might conflict with the invitation PR if it is merged in first...

Reply:
I'll make sure to merge yours first since it is higher priority.